internal/cmd/configssh.go

```diff
@@ -215,3 +216,12 @@ func readStr(filename string) (string, error) {
 	}
 	return string(contents), nil
 }
+
+func writeSSHUXState(ctx context.Context, client *coder.Client) {
+	// Update UXState that coder config-ssh has been run by the currently
+	// authenticated user
+	err := client.MyUXState(ctx, map[string]interface{}{"cliSSHConfigured": true})
+	if err != nil {
+		fmt.Println("⚠ The Coder web client may not recognize that you've configured SSH.")
```

Comment:
`clog.LogWarn` could provide some nice formatting for this.

Reply:
Awesome stuff :+1: